src/test/resources/Features/OCD-2023-monitorDownloadFiles.feature

```diff
@@ -30,10 +30,17 @@ Feature: OCD-2023: Monitor that download files on CHPL download resources page a
       | PROD |    2015 | xml  |    1 |  150 | MB    |   590 |
       | PROD |    2015 | csv  |    1 |  350 | KB    |   590 |
 
-  Scenario: At any given time, when downloaded, Surveillance Activity file is up-to-date
-    When I download the Surveillance Activity file
-    Then the downloaded file shows surveillance-all.csv filename
-
-  Scenario: At any given time, when downloaded, Non-Conformities file is up-to-date
-    When I download the Non-Conformities file
+ Scenario: At any given time, when downloaded, Non-Conformities file is up-to-date
+    When I select the Non-Conformities file from drop-down
+    And I download "surveillance-with-nonconformities" file
     Then the downloaded file shows surveillance-with-nonconformities.csv filename
```

Comment:
This file should also have the  yyyymmdd extension. At the end of this ticket, _all_ of the files in that download dropdown should